packages/core/src/input/input.element.scss

```diff
@@ -107,5 +107,6 @@
 }
 
 :host([_disabled]) {
-  --border-color: #{$cds-alias-status-disabled};
+  --border-color: #{$cds-alias-object-border-color-tint};
```

Comment:
Could we make the border color here the same as the text color on the next line? `$cds-alias-status-disabled`

Right now, the text is of a disabled input is darker than the border and it looks inconsistent.

Reply:
```
I think this looks wrong: the search border should definitely be darker than the select border.

What do you think?
```

I see what you are saying. I say keep it as you have it now. But I think this has uncovered a deeper visual issue in general that we won't solve right here and right now.

Pinging @colinreedmiller so that this is on his radar for future consideration.